packages/tools/devtools/devtools-browser-extension/webpack.test.js

```diff
@@ -46,6 +47,9 @@ module.exports = (env) => {
 			new webpack.ProvidePlugin({
 				process: "process/browser",
 			}),
+			new Dotenv({
+				path: "./dev-example.env",
```

Comment:
I'd say let's remove the custom file entirely, remove the `path` option here, and add `systemvars: true` (like for the production webpack config) and add documentation in the README of the package to clarify that if we want some token to be used when building+bundling locally, it should be set in an environment variable `DEVTOOL_TELEMETRY_TOKEN`.

Reply:
Nit: what do you both think about calling this `DEVTOOLS_TELEMETRY_TOKEN` instead? We generally use "devtools" rather than "devtool", so it would make it a bit more consistent.

Reply:
Yeah, I agree with that rename.

Reply:
Renamed it